src/features/defi/components/Overview/UnderlyingAssetsTags.tsx

```diff
@@ -0,0 +1,49 @@
+import { Tag } from '@chakra-ui/react'
+import React from 'react'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+
+import { PairIcons } from '../PairIcons/PairIcons'
+import type { AssetWithBalance } from './Overview'
+
+type UnderlyingAssetsTagProps = {
+  asset: AssetWithBalance
+  showPercentage?: boolean
+  children?: React.ReactNode
+}
+
+type UnderlyingAssetsTagsProps = {
+  underlyingAssets: AssetWithBalance[]
+  showPercentage?: boolean
```

Comment:
We default this to `true` already, so it's always defined.

Reply:
Unfortunately tsc isn't so smart in figuring out that the component defaults the destructured value and thus it's always defined, so we will need to keep it optional, so the component doesn't ask for it to be passed.
With that being said, it does seem more ergonomic with `false` as default -  be948d9df322aedad425710ee14c267b82bfcaef